exts/cesium.omniverse/mdl/cesium.mdl

```diff
@@ -17,6 +17,10 @@ annotation annotation_not_connectable();
 
 export const auto DEFAULT_NULL_FEATURE_ID = -1;
 
+
+// Should match MAX_IMAGERY_LAYERS_COUNT in FabricMaterial.cpp
+export const auto MAX_IMAGERY_LAYERS_COUNT = 16;
```

Comment:
Is it possible to have this in a unified location?

Reply:
I could remove the constant and change the code below to

```
    gltf_texture_lookup_value[] imagery_layers(
        imagery_layer_0,
        imagery_layer_1,
        imagery_layer_2,
        imagery_layer_3,
        imagery_layer_4,
        imagery_layer_5,
        imagery_layer_6,
        imagery_layer_7,
        imagery_layer_8,
        imagery_layer_9,
        imagery_layer_10,
        imagery_layer_11,
        imagery_layer_12,
        imagery_layer_13,
        imagery_layer_14,
        imagery_layer_15,
    );
```

Only concern would be forgetting to update the C++ side if the MDL side changes, or vice-versa.

Reply:
I'll make this change and just leave a comment :+1:

Reply:
https://github.com/CesiumGS/cesium-omniverse/pull/552/commits/c586cc4b7524276ab71ba545242986073b1ac4ef